### PR TITLE
Instancio bootstrapping agreement, 2024-05-06

### PR DESCRIPTION
**Pull request by armandino**

### CODE REVIEW

File: PROJECTS.yaml [hunk 1]
```diff
@@ -73,3 +73,13 @@ openrewrite:
   description: >
     Automate the refactoring of your Java codebase. OpenRewrite offers scalable, safe, and idempotent
     code transformations to modernize and maintain your applications.
+
+openrewrite:
```
Comment (evanchooly) [inline PROJECTS.yaml:77]: this needs to be updated

Comment (timtebeek) [reply PROJECTS.yaml:77]:
```suggestion
instancio:
```

Comment (armandino) [reply PROJECTS.yaml]: Updated. thanks!

**Review by ebullient — APPROVED** [review]

**Comment by ebullient** [comment]
Hi Arman,

Thank you for your interest in having Instancio join the Commonhaus Foundation. We truly appreciate your support and willingness to put this PR together.

As we are in the bootstrapping phase (about 12 months), we are focusing on more established projects. Please consider re-applying once this phase is complete. We can rebase and re-open this PR at that time.

We hope you will continue to participate as a foundation member over the coming months. This foundation needs members as much as it needs projects to be successful.

Thank you for your understanding. We hope to see you around.

🙏 🫂

**Pull request closed without merging**